package.json

```diff
@@ -22,14 +22,12 @@
   },
   "homepage": "https://github.com/SparkPost/nodemailer-sparkpost-transport",
   "dependencies": {
-    "dotenv": "^1.2.0",
-    "email-addresses": "^2.0.1",
-    "sparkpost": "^1.1.0"
+    "sparkpost": "^1.3.5"
```

Comment:
Do we mean to force people to keep current with node-sparkpost?  Honestly not sure of our policy.